### DISCUSSION TITLE
Welcome to Khoros Python API Discussions!

### DISCUSSION THREAD

**Discussion by jeffshurtliff** [jeffshurtliff/khoros#22] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ruckuseduops** [comment]
Hi Jeff,

Just starting with the khoros package (super helpful!) and looking at Bulk Data v2 API and the doc section for example is blank. Looked through the package and didn't see anything specific in terms of classes or methods. Just wondering how to configure the fromdate and todate in the call. I can use Khoros Python3 example but would be nice to do it all within your package.

**Reply by jeffshurtliff** [reply]
Hi @ruckuseduops,

Interestingly enough, I've been working on several enhancements for the Bulk Data API within the package that will be released in [version 5.2.0](https://khoros.readthedocs.io/en/beta/changelog.html), which I hope to release this week.

Part of that release includes new documentation for using the Bulk Data API, which you can already view [here](https://khoros.readthedocs.io/en/beta/bulk-data-api.html#querying-the-bulk-data-api).

Hope this helps!
Jeff
